README.md

```diff
@@ -52,6 +52,7 @@ dokku postgres:create lolipop
 # official postgres image
 export POSTGRES_IMAGE="postgres"
 export POSTGRES_IMAGE_VERSION="9.4.4"
+export POSTGRES_CUSTOM_ENV="USER=alpha;HOST=beta"
```

Comment:
They were intended just to be examples. I'm totally open to something different maybe `EXAMPLE_ENV=VALUE`?